ts/components/InstabugButtons.tsx

```diff
@@ -93,6 +94,7 @@ class InstabugButtonsComponent extends React.PureComponent<Props, State> {
             this.state.instabugReportType.value,
             dismiss
           );
+          this.props.dispatchUpdateInstabugBadge();
```

Comment:
Technically you are not updating instabug badge, instead you are loading unread messages counter
```suggestion
          this.props.dispatchLoadInstabugUnreadMessagesCounter();
```

Reply:
Yes, I'm reloading unread messages counter